catroid/src/main/res/layout/landing_page.xml

```diff
@@ -26,68 +26,100 @@
 
-<RelativeLayout
-    android:layout_width="match_parent"
-    android:layout_height="match_parent">
-    <androidx.recyclerview.widget.RecyclerView
-        android:paddingStart="16dp"
-        android:paddingEnd="0dp"
-        android:id="@+id/recycler_view"
-        android:layout_marginTop="24dp"
-        android:layout_marginBottom="48dp"
-        android:layout_width="match_parent"
-        android:layout_height="match_parent"
-        android:overScrollMode="never"
-        app:layoutManager="LinearLayoutManager">
-    </androidx.recyclerview.widget.RecyclerView>
+                    <ImageView
+                        android:id="@+id/upload_button"
+                        android:contentDescription="@string/main_menu_upload"
+                        android:layout_width="wrap_content"
+                        android:layout_height="wrap_content"
+                        android:layout_alignParentBottom="true"
+                        android:layout_alignParentEnd="true"
+                        android:layout_marginBottom="12dp"
+                        android:layout_marginEnd="12dp"
+                        android:src="@drawable/ic_publish_circle" />
+
+                </RelativeLayout>
+
+                <TextView
+                    android:id="@+id/my_projects"
+                    android:text="@string/main_menu_programs"
+                    android:layout_width="match_parent"
+                    android:layout_height="wrap_content"
+                    android:paddingTop="16dp"
+                    android:paddingBottom="16dp"
+                    android:paddingStart="20dp"
+                    android:paddingEnd="32dp"
+                    android:textColor="@color/landing_page_icons"
+                    android:drawableEnd="@drawable/ic_arrow_forward_vector"
+                    android:clickable="true"
+                    android:focusable="true"
+                    android:textSize="?attr/x_large"
+                    android:background="@drawable/button_background_selector" />
 
-    <com.google.android.material.floatingactionbutton.FloatingActionButton
+                <androidx.recyclerview.widget.RecyclerView
+                    android:id="@+id/my_projects_recyclerview"
+                    android:paddingStart="14dp"
+                    android:paddingEnd="0dp"
+                    android:clipToPadding ="false"
+                    android:layout_width="match_parent"
+                    android:layout_height="wrap_content"
+                    app:layoutManager="LinearLayoutManager"
+                    android:orientation="horizontal" />
+
+                <androidx.recyclerview.widget.RecyclerView
+                    android:id="@+id/recycler_view"
+                    android:layout_marginStart="4dp"
+                    android:layout_marginTop="8dp"
+                    android:layout_width="match_parent"
+                    android:layout_height="180dp"
```

Comment:
in case the btns height gets changed in the future.
```suggestion
                    android:layout_height="wrap_content"
```

Reply:
This does not work, because if the other recyclerview is empty on creation, the size of this one gets all messed up.
This is why I used a fixed size for the recyclerview, which will be deleted anyway in the next issue. 
This is why I used a scrollview already in this issue. On older phones the buttons would be so small because the recyclerview does not have enough space, so I used a fixed size and a scroll view